### ISSUE TITLE
App takes screen shots automatically when using third-party Screenshot app?

### ISSUE THREAD

**Issue by sunstarjeff** [responsively-org/responsively-app#643]
When using a third-party screen shot app, Responsively starts taking screenshots automatically of all active panes.

Also, I don't see any setting to turn off automatic screen shots or adjust hotkeys related to screen shots.

![image](https://user-images.githubusercontent.com/14909807/143310348-bf51e640-da89-4f4c-964d-91d588a513d4.png)

# 🐞 bug report

### 🌍 Your Environment

<pre><code>

</code></pre>

### 🔥 Exception or Error or Screenshot

<pre><code>

</code></pre>

**Comment by jjavierdguezas** [comment]
Are you triggering your third-party screenshot app using a keyboard shortcut? If so, is it the same shortcut used in Responsively (`Cmd/Ctrl+PrtSc` or `Cmd/Ctrl+Shift+S`)?

**Comment by sunstarjeff** [comment]
Yes.  Is there a mechanism to change the hot key / shortcut used in Responsively?

**Comment by jjavierdguezas** [comment]
> Yes. Is there a mechanism to change the hot key / shortcut used in Responsively?

Right now there is no such mechanism 😓... It could be a good feature to add (or at least deactivate some of them)
cc: @manojVivek

**Comment by jjavierdguezas** [comment]
closing this and opening a feature request on #745

**Comment by 202111023** [comment]
@sunstarjeff I tried recreating the mentioned issue, but it's working fine with me. I tried taking screenshots, but the problem did not occur to me. Responsively uses these shortcuts to take screenshots, "ctrl + shift + s" or "ctrl + prtsc", and possibly the screenshot hotkey for your 3rd party app aligns with the shortcut for Responsively's screenshot feature.
Using the screenshot shortcut keys takes a screenshot of all the devices active in Responsively window.
